proto/minder/v1/minder.proto

```diff
@@ -2307,6 +2307,8 @@ message Invitation {
     // project_display is the display name of the project to which the user
     // is invited.
     string project_display = 10;
+    // inviteURL is the URL that can be used to accept the invitation.
+    string invite_url = 11;
```

Comment:
I haven't been following this feature too closely, but why is the invite URL configurable at all? Shouldn't this be something set by the server?

Reply:
I think both are true, because the invite URL is generated on the backend side, but it is based on where the FE server is. I decided to pass this from the environment through the server config. Is this what you meant?

Reply:
What I meant to ask is, is the invite URL something the user can affect or is it just given to the user in the reply? If it's just in the reply and the user can't change the invite base URL when they create the invite, then it's OK

Reply:
Ah, I see. No, the user has no control over the invite URL and how it's constructed 👍